src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './Calculator/calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
```

Comment:
- [ ] Kindly make the `App.js` a class based component aswell, since the type of component you follow usually has to be uniform throughout the project.

Reply:
Thanks for the review!. 👍